numba/tests/test_loopdetection.py

```diff
@@ -1,47 +0,0 @@
-from __future__ import print_function
```

Comment:
Note these tests were superseded a long time ago by the CFG tests in `test_flow_control.py`.